azure/storagemanager/testspecs/azure-storage-decision-matrix.yaml

```diff
@@ -87,6 +87,28 @@ rows:
           priority: 2
           thin_provisioning: false
           drive_type: "Premium_LRS"
+        - min_iops: 18000
```

Comment:
The previous row should now be
```
        - min_iops: 16000
          max_iops: 16000
          instance_type: "*"
          instance_max_drives: 8
          instance_min_drives: 1
          region: "*"
          min_size: 8192
          max_size: 16384
```
and so on... 

the last row will now be
```
        - min_iops: 20000
          max_iops: 20000
          instance_type: "*"
          instance_max_drives: 8
          instance_min_drives: 1
          region: "*"
          min_size: 32768
          max_size: 32768
          priority: 2
          thin_provisioning: false
          drive_type: "Premium_LRS"
```

Reply:
@adityadani Do you want me to follow that pattern for other disk types too? They haven't been following it previously.